index.bs

```diff
@@ -2708,6 +2715,121 @@ See [[#interface-object]] for details on how a [=constructor operation=] is to b
 <div data-fill-with="grammar-ArgumentNameKeyword"></div>
 
 
+<h4 id="idl-stringifiers">Stringifiers</h4>
+
+When an [=interface=] has a <dfn id="dfn-stringifier">stringifier</dfn>, it indicates that objects
+that implement the interface have a non-default conversion to a string.  As mentioned above,
+stringifiers can be specified using a <emu-t>stringifier</emu-t> keyword.
```

Comment:
Removing "As mentioned above" since I think it refers to the table in special operations section.

```suggestion
that implement the interface have a non-default conversion to a string.  Stringifiers can be
specified using a <emu-t>stringifier</emu-t> keyword, which creates a stringifier operation when
used alone.
```